src/components/20-molecules/dropdown/CHANGELOG.md

```diff
@@ -20,15 +20,15 @@
   component consumer had previously assumed undefined or uninitialized
   behaviour.
 
-## Migrating from version 2 to 3
+## Version 3.2.3
 
-- The attribute `valid` was renamed to `checkmark` (respectively to the camel-case `checkMark` property used e.g. under React).
```

Comment:
Did you remove this part on purpose?

Reply:
It was moved, but not removed.  It's still there on row 34. The order of the records seemed to be false.